PRODUCTION.md

```diff
@@ -0,0 +1,82 @@
+# Using S3 for continuous integration testing with Docker
+
+When you start the Scality S3server image, you can adjust the configuration of
+the Scality S3server instance by passing one or more environment variables
+on the docker run command line.
+
+## Environment Variables
+
+### HOST_NAME
+
+This variable specifies an host name:
+
+`docker run -d --name s3server -p 8000:8000 -e HOST_NAME=new.host.com
+scality/s3server`
+
+### ACCESS_KEY and SECRET_KEY
+
+These variables specify a Scality S3server user credentials named Docker:
```

Comment:
These variables specify authentication credentials for an account named "Docker".  You can set credentials for many accounts by editing conf/authdata.json (see below for further info), but if you just want to specify one set of your own, you can use these environment variables.